include/seccomp.h.in

```diff
@@ -66,6 +67,7 @@ enum scmp_filter_attr {
 	SCMP_FLTATR_API_TSKIP = 5,	/**< allow rules with a -1 syscall */
 	SCMP_FLTATR_CTL_LOG = 6,	/**< log not-allowed actions */
 	SCMP_FLTATR_SPEC_ALLOW = 7, /**< disable SSB mitigation */
+	SCMP_FLTATR_NEW_LISTENER = 8,   /**< returns a listener fd */
```

Comment:
See my previous comment regarding attribute names (my apologies once again).  Similar to the last one, this should probably be `SCMP_FLTATR_CTL_<blah>`.